assets/js/modules/analytics-4/datastore/report.js

```diff
@@ -110,6 +111,38 @@ const fetchGetReportStore = createFetchStore( {
 	},
 } );
 
+const gatheringDataStore = createGatheringDataStore( 'analytics-4', {
+	dataAvailable:
+		global._googlesitekitModulesData?.[ 'data_available_analytics-4' ],
+	determineDataAvailability: createRegistrySelector( ( select ) => () => {
```

Comment:
Seeing this in use now, I think `selectDataAvailability` might be more apt for the name here which makes it clear that the value is a selector function.